src/composables/createMultisig.ts

```diff
@@ -0,0 +1,89 @@
+import { ref } from '@vue/composition-api';
```

Comment:
If the composable function is `useMultisig` the filename should be `multisig.ts` instead of `createMultisig.ts`.

Reply:
Right. It's a temporary workaround so that the rebasing for Nikita would be easy. In his PR, he has created 'useMultisig` already. We are gonna move everything from `createMultisig` to 'multisig.ts` file.

Reply:
I renamed it as everyone is gonna use it.